File: Core/Entities/WeatherSummary.cs
using System;
using System.Collections.Generic;
using Core.Identity.Entities;

namespace Core.Entities
{
    public class WeatherSummary : BaseEntity
    {
        public string IpAddress { get; set; }
        public int? UserSearchId { get; set; }
        public UserSearch UserSearch { get; set; }
        public string Title { get; set; }
        public string LocationType { get; set; }
        public int WoeId { get; set; }
        public string LatitudeLongitude { get; set; }
        public string TimeZone { get; set; }
        public string Time { get; set; }
        public string  SunRise { get; set; }
        public string SunSet  { get; set; }
        public List<Weather> ConsolidatedWeather { get; set; }
        public LocationSearchResult Parent { get; set; }

        public List<Source> Sources { get; set; }



        
    }
}

File: Client/src/app/shared/models/weatherSummary.ts
import { Consolidated_Weather } from "./consolidated_Weather";
import { LocationInfo } from "./locationInfo";
import { Source } from "./source";

export class WeatherSummary{
    title: string;
    location_Type: string;
    woeId: number;
    latLong: string;
    timeZone: string;
    time: Date;
    sun_Rise: Date;
    sun_Set: Date;
    time_Zone: string
    consolidated_Weather: Consolidated_Weather[];
    parent: LocationInfo;
    sources: Source[];
    
}